interpreter/src/main/scala/sigmastate/crypto/Platform.scala

```diff
@@ -1,12 +1,17 @@
 package sigmastate.crypto
 
 import org.bouncycastle.crypto.ec.CustomNamedCurves
-import org.bouncycastle.math.ec.{ECFieldElement, ECPoint}
+import org.bouncycastle.math.ec.{ECPoint, ECFieldElement, ECCurve}
 
 import java.math.BigInteger
 
 /** JVM specific implementation of crypto methods*/
 object Platform {
+  /** Returns a [[Curve]] instance describing the elliptic curve of the point p
```

Comment:
No need to replicate in the comments what is already in the code, namely, "Returns a [[Curve]] instance".

"Description of elliptic curve of point p which belongs to the curve" is enough

Reply:
Fixed, but this is a matter of taste. "Returns ..." is ok accoding to the style guide.

Reply:
It is not about "Returns...", rather, "a [[Curve]] instance"